Mathlib/SetTheory/Ordinal/Notation/FundamentalSequence.lean

```diff
@@ -0,0 +1,186 @@
+/-
+Copyright (c) 2024 Violeta Hernández Palacios. All rights reserved.
+Released under Apache 2.0 license as described in the file LICENSE.
+Authors: Violeta Hernández Palacios
+-/
+import Mathlib.Data.Set.Operations
+
+/-!
+# Fundamental sequences
+
+If `x` is a successor limit in a linear order `α`, a fundamental sequence for `x` is a strictly
+monotonic sequence `ℕ → α` whose supremum is `x`.
+
+If `x` is not a successor limit, we can define two other types of fundamental sequences: the empty
+sequence `∅` with limit `⊥`, and the singleton sequence `{x}` with limit `succ x` (the limit here is
+more properly the least strict upper bound). This ensures that every countable ordinal
```

Comment:
and my comment above justifies why this needs to be `{x}` and not `{succ x}` (which confused me initially)